src/Sulu/Bundle/MediaBundle/Entity/Collection.php

```diff
@@ -88,12 +88,12 @@ class Collection implements CollectionInterface, PermissionInheritanceInterface
     private $key;
 
     /**
-     * @var DoctrineCollection<int, CollectionMeta>
+     * @var DoctrineCollection<array-key, CollectionMeta>
```

Comment:
this are always `int` as it is not indexed by something else in the mapping. in case all our collections should be not indexed by something else as that would be a performance issue.

Reply:
Yes, by default doctrine collections are indexed by ints but in general it could be anything (according to doctrine). Reverting it back to ints upsets the phpstan again, since it can be `array-key` which is just an alias to `string|int`.

Reply:
yeah but would go with what it really is `int` here and not let phpstan decide what it isnt ;)

Reply:
```suggestion
     * @var DoctrineCollection<int, CollectionMeta>
```

Still go with int here, phpstan can currently not detect it, but it is always an int.